### DISCUSSION TITLE
Selections and ride highlights should be visible to other players in multiplayer

### DISCUSSION THREAD

**Discussion by Shadowblitz16** [OpenRCT2/OpenRCT2#16850] [Feature requests] [answered]
Tile selections and the white ride/scenery highlights when your about to place down a ride should be visible to other players in multiplayer.
This makes it easier to see what other people are doing and can be used to communicate where are good places to put rides down.

**Comment by duncanspumpkin** [comment, marked as answer]
Super impossible to do atm due to how everything works but it is something we do eventually want to do.

**Reply by Shadowblitz16** [reply]
k cool

**Comment by IntelOrca** [comment]
The first thing that comes to mind is, what happens if two players overlap their ghost elements? Currently the game searches for ghost elements of the same type to remove for most actions - so it may end up removing the ghost of another player.

**Reply by Shadowblitz16** [reply]
I would guess that yours would take priority and would render itself over the other player's

**Reply by Shadowblitz16** [reply]
Maybe even a different color could be used for other player's ghost elements like light blue?

**Reply by Broxzier** [reply]
How should the game know which ghost elements belong to you or other players? Besides sending the ghost elements over, knowing this is also a technical limitation. How ghosts are handled should be made separate entirely, and that's a huge refactoring job. Adding support for different colours is the least difficult then.
